backend-models/service-tests/showDuggaservice-test.txt

```diff
@@ -0,0 +1,666 @@
+                            "UNK" }; 
+
+
+
+SQL-query:  INSERT INTO listentries(cid,entryname,link,kind,pos,creator,visible,vers,moment,gradesystem,highscoremode,feedbackenabled,feedbackquestion) 
+            VALUES ($preValuesListentries[0], $preValuesListentries[1], $preValuesListentries[2], $preValuesListentries[3], $preValuesListentries[4], $preValuesListentries[5], $preValuesListentries[6], $preValuesListentries[7], $preValuesListentries[8], $preValuesListentries[9], $preValuesListentries[10],$preValuesListentries[11],$preValuesListentries[12]);
+
+$pre-values =
+{
+    9999,
+    “dfg4zxc5”,
+    “asdfasdf”,
+    5019
+};
+SQL-query: INSERT INTO userAnswer(cid,hash,password,moment) VALUES ($pre-values[0], $pre-values[1], $pre-values[2], $pre-values[3]);
+
+***
+* send parameters input to showDuggaservice.php
+***
+send
+{
+	$hash = “dfg4zxc5”
+}
+
+***
+Gather service output
+***
+Save the values of the echoed array as a JSON (at the end of the file. echo json_encode($array);)
+This is the expected output for the micro service.
+
+***
+* Remove inserted rows from DB
+***
+
+SQL-query: DELETE FROM userAnswer ORDER BY aid DESC LIMIT 1;
+DELETE FROM listentries WHERE cid = 9999;
+DELETE FROM course WHERE cid = 9999;
+—------------------------------------------------------------------------------------------------------------------
+
+—------------------------------------------------------------------------------------------------------------------
+TEST #8 - 327 - 334
+super-view data from useranswer on moment
+
+***
+Pre-req: isSuperUser = TRUE
+	  $hash != “UNK”
+	  $rows == NULL
+***
+Pre-query:
+
+$preValuesCourse = {	9999
+			1
+};
+
+INSERT INTO course(cid, creator) VALUES ($preValuesCourse[0], $preValuesCourse[1]);
+
+$preValuesListentries = {   9999, 
+                            "Inserttobedeleted",
+                            "UNK",
+                            4,
+                            12,
+                            2,
+                            1,
+                            1337,
+                            5019,
+                            1,
+                            1,
+                            0, 
+                            "UNK" }; 
+
+SQL-query:  INSERT INTO listentries(cid,entryname,link,kind,pos,creator,visible,vers,moment,gradesystem,highscoremode,feedbackenabled,feedbackquestion) 
+            VALUES ($preValuesListentries[0], $preValuesListentries[1], $preValuesListentries[2], $preValuesListentries[3], $preValuesListentries[4], $preValuesListentries[5], $preValuesListentries[6], $preValuesListentries[7], $preValuesListentries[8], $preValuesListentries[9], $preValuesListentries[10],$preValuesListentries[11],$preValuesListentries[12]);
+
+SELECT * FROM userAnswer WHERE hash = “xxxxxxxx”// This should return nothing, otherwise, change hash to something where previous query returns nothing.
+
+***
+* send parameters input to showDuggaservice.php
+***
+send
+{
+	$moment = 5019
```

Comment:
make sure you have the right value here, moment will by this point have changed value.